jenkins/build.sh

```diff
@@ -68,7 +67,10 @@ cd $BASE_DIR
 echo "Using MAJOR_VERSION=${MAJ_VER} MINOR_VERSION=${MIN_VER} REVISION=${PATCH_VER} BUILD=${BUILD}"
 version="${MAJ_VER}.${MIN_VER}.${PATCH_VER}"
 
+rm -rf ${BASE_DIR}/dist; mkdir -p ${BASE_DIR}/dist/temp
```

Comment:
dist is created by setuptools. Not a good idea to use same for our purpose.
Can we create another directory with name .backup in base directory and copy the file there. After restoring, we can delete this directory too.

Reply:
Updated